Check if last tested and minimum NVDA versions exist in available API versions (#17)
nvaccess/addon-datastore-validation: README.md

  - The submission addonName matches the manifest summary field
  - The submission description matches the manifest description field
  - The homepage URL matches the manifest URL field

>   - The addon versions match, and last tested & minimum required versionsexist in NVDA API versions.

Reviewer: add space

```suggestion
  - The addon versions match, and last tested & minimum required versions exist in NVDA API versions.
```

>   - The addon versions match, and last tested & minimum required versions exist in NVDA API versions.

Reviewer: can this be on a new line. 
```suggestion
  - The addon versions match
  - The last tested & minimum required versions are valid NVDA API versions.
```
